feedback-module/src/assets/constants2.js

```diff
@@ -0,0 +1,79 @@
+const MISSING_INFO = {
```

Comment:
This file is temporary for now. I just kept it with the oldest formatting of the constants file just so the pages I wasn't refactoring wouldn't break. We will need to delete it before the merge is made